packs/data/feats.db/viking-dedication.json

```diff
@@ -13,7 +13,7 @@
             "value": null
         },
         "description": {
-            "value": "<p>Vikings spend long periods of time at sea, only to leap from their boats at a moment's notice, charge through the surf, and take their enemies by storm. You're trained in Sailing Lore. When moving through water using your land Speed, you ignore difficult terrain resulting from shallow water, such as in bogs or on beaches. In addition, if you roll a success for your Athletics check to @Compendium[pf2e.actionspf2e.Swim]{Swim}, you get a critical success instead.</p>\n<p><strong>Special</strong> You can't select another dedication feat until you have gained two other feats from the viking archetype.</p>"
+            "value": "<p>Vikings spend long periods of time at sea, only to leap from their boats at a moment's notice, charge through the surf, and take their enemies by storm. You're trained in Sailing Lore. When moving through water using your land Speed, you ignore difficult terrain resulting from shallow water, such as in bogs or on beaches. In addition, if you roll a success for your Athletics check to @Compendium[pf2e.actionspf2e.Swim]{Swim}, you get a critical success instead.</p>\n<p><strong>Special</strong> You can't select another dedication feat until you have gained two other feats from the @UUID[Compendium.pf2e.journals.vx5FGEG34AxI2dow.JournalEntryPage.2CbiV0VtDKZfQNte]{Viking}g archetype.</p>"
```

Comment:
`{Viking}g`

The extra `g` is for the greataxe they carry.

Reply:
Definitely. (Wonder how I managed to do that?)